exercises/pangram/src/main/java/Pangrams.java

```diff
@@ -0,0 +1,6 @@
+public class Pangrams {
```

Comment:
Thanks for taking the time to explain that, that's very useful! :)

I think I would prefer it to be a Function Object. That's mostly because if we have a "sentence" object I wouldn't expect it's only function to be "is it a pangram", that's not the first thing I associate with a sentence. So it doesn't really feel like it encapsulates what a sentence is, any more than a string does. We only really want that one function so I think it makes more sense to make it a Function Object. 

I also think that it makes it more clear to the person doing the exercise what the exercise is about. I know the README explains that but it's nice if everything is as clear as possible :) 

But I'm open to being convinced otherwise if you disagree :)